components/Charts/TransactionVolume.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from "react";
+import mean from "ramda/src/mean";
+
+import GeneralChart, { GeneralChartProps }  from "./GeneralChart";
+import ChartBox from "./ChartBox";
+import Metric from "types/domain/Metric";
+
+const TransactionVolume: FC<Pick<GeneralChartProps, "data">> = ({ data }) => {
+  const valueAccessor = (d: Metric) => d.transactions_count;
```

Comment:
I dunno how well TS will play, but you can use `ramda/src/prop` and do:
```suggestion
  const valueAccessor = prop('transactions_count')
```